testplan/runnable/base.py

```diff
@@ -573,9 +573,7 @@ def add_remote_service(self, remote_service: RemoteService):
         remote_service.start()
 
     def _stop_remote_services(self):
-
-        for name, rmt_svc in self.remote_services.items():
-            self.logger.info("Stopping Remote Server %s", name)
```

Comment:
why remove this logging

Reply:
rmt_svc will log a message inside its `stop` method, i think it's kinda duplicate here